SMLHelper/Patchers/WaterBiomeManagerPatcher.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using HarmonyLib;
+using System.Reflection.Emit;
+namespace SMLHelper.V2.Patchers
+{
+    internal class WaterBiomeManagerPatcher
+    {
+            [HarmonyPatch(typeof(WaterBiomeManager), nameof(WaterBiomeManager.Start))]
+            [PatchUtils.Prefix]
+            internal static bool WaterBiomeManager_Start_Prefix(WaterBiomeManager __instance)
+            {
+                for (var i = 0; i < BiomeThings.Variables.Biomes.Count; i++)
+                {
+                    var biome = BiomeThings.Variables.Biomes[i];
+                    var settings = new WaterBiomeManager.BiomeSettings()
+                    {
+                        name = biome.BiomeName,
+                        settings = biome.WaterScapeSettings,
+                        skyPrefab = new UnityEngine.GameObject("Useless Junk.")
```

Comment:
Please dont implicitly add "useless junk" skies. Consumers must have full control over setting up their skies so there should be a `SetupBiomeSettings` method in the `Biome` class, then we add them to the `WaterBiomeManager.biomeSettings`, `WaterBiomeManager.biomeLookup` and `WaterBiomeManager.skies` here.

Reply:
The useless junk sky is almost immediately destroyed and replaced with a provided sky by the consumer. I only add the useless junk sky as to pass a check.